### DISCUSSION TITLE
The cheatsheet is really ugly and unreadable !

### DISCUSSION THREAD

**Discussion by tmiele** [raysan5/raylib#2930] [Ideas]
Hello,

Can you make the official [cheatsheet](https://www.raylib.com/cheatsheet/cheatsheet.html) more readable : 

- The grey background with fade black text is a mess.
- And more i test the zoom on chrome and the cheatcheat not scale correctly and go outside of screen at right.
- This is the only documentation you give... Can you make it digest for eyes please.

![Capture d’écran 2023-02-20 065916 ](https://user-images.githubusercontent.com/2271125/220021968-01bad565-00a0-41ed-ae25-e37f380071f8.png "cheatcheat not scale correctly")

Thanks, I test Raylib-cslo

**Comment by raysan5** [comment]
Thanks for your feedback, no plans to change it for the moment.
